website/static/js/nodesDelete.js

```diff
@@ -280,24 +282,105 @@ NodesDeleteViewModel.prototype.back = function() {
     this.page(this.SELECT);
 };
 
-function NodesDelete(selector, node) {
+/**
+ * view model which corresponds to nodes_delete.mako (#nodesDelete)
+ * Used for quickly deleting nodes with no children.
+ * @type {QuickDeleteViewModel}
+ */
+
+var QuickDeleteViewModel = function(nodeType, isPreprint, nodeApiUrl){
     var self = this;
+    self.confirmationString = $osf.getConfirmationString();
+    self.QUICKDELETE = 'quickDelete';
+    self.SELECT = 'select';
+    self.CONFIRM = 'confirm';
+    self.page = ko.observable(self.QUICKDELETE);
+    self.confirmInput = ko.observable('');
+    self.canDelete = ko.observable(false);
+    self.nodeType = nodeType;
+    self.nodeApiUrl = nodeApiUrl;
+    self.isPreprint = isPreprint;
+    self.message = ko.computed(function(){
+        var preprintMessage = '<br><br>This ' + self.nodeType + ' contains a <strong>preprint</strong>. Deleting this ' +
```

Comment:
Can we consolidate some of the messages that are identical on the `QuickDeleteViewModel` and `NodesDeleteViewModel`?